src/python/grpcio_tests/commands.py

```diff
@@ -193,6 +193,7 @@ class TestGevent(setuptools.Command):
         'unit._server_ssl_cert_config_test',
         # TODO(https://github.com/grpc/grpc/issues/14901) enable this test
         'protoc_plugin._python_plugin_test.PythonPluginTest',
+        'protoc_plugin._python_plugin_test.SimpleStubsPluginTest',
```

Comment:
Why is it banned for `gevent`?

Reply:
[The other test in this file also doesn't work.](https://github.com/grpc/grpc/blob/d8623ba548cc9d65d98bf136d1bd52b7aba1c219/src/python/grpcio_tests/commands.py#L195) After seeing that the failure was due to the setup in the rest of the file, I didn't debug it further.